frontend/src/App.js

```diff
@@ -1,5 +1,16 @@
+import { Route, Switch } from 'react-router-dom'
+import PlantListPage from './plantlistpage/PlantListPage'
+
 function App() {
-  return <div className="App"></div>
+  return (
+    <div>
```

Comment:
What do you need the div for? Isn't it possible to wrapp the whole thing wiht just the Switch element?

Reply:
Gonna test it 👍